build_script.py

```diff
@@ -92,7 +92,6 @@ def main():
 
     sourceFiles = [
                    "XCTAssert.swift",
-                   "XCTestCaseProvider.swift",
                    "XCTestCase.swift",
                    "XCTimeUtilities.swift",
                    "XCTestMain.swift",
```

Comment:
I've been meaning to change this to a `glob.glob()` call. That way we won't have to modify this list of files every time.